src/java/org/apache/cassandra/io/FSDiskFullWriteError.java

```diff
@@ -18,16 +18,22 @@
 
 package org.apache.cassandra.io;
 
+import java.io.File;
+import java.io.IOException;
+
 public class FSDiskFullWriteError extends FSWriteError
 {
-    public FSDiskFullWriteError(Throwable cause, String path)
+    public FSDiskFullWriteError(String keyspace, long mutationSize)
     {
-        super(cause, path);
+        super(new IOException(String.format("Insufficient disk space to write %s bytes into the %s keyspace",
+                                            mutationSize,
+                                            keyspace)),
+              new File(""));
```

Comment:
Nit: Both `FSDiskFullWriteError` and `FSNoDiskAvailableForWriteError` call the super constructor of `FSWriteError` with `new File("")`. Maybe we could add a new `FSWriteError` constructor without the file argument.